src/config/cli.rs

```diff
@@ -39,6 +40,16 @@ pub struct Cli {
     /// Port to listen on (default: 8011).
     pub port: Option<u16>,
 
+    /// The hosts the server will listen on.
+    #[arg(
+        long,
+        value_name = "IP_ADDR",
+        default_value = "127.0.0.1",
+        value_delimiter = ',',
+        help_heading = "Network Options"
+    )]
+    pub host: Vec<IpAddr>,
```

Comment:
Comparing this to upstream anvil I have noticed that we don't match env vars they use. Is there a reason for that?

Reply:
Not intentionally left out. Will add it so theres full parity. 🙏

Reply:
Updated ✅